.github/workflows/DeployToDev.yml

```
@@ -0,0 +1,77 @@
+          strip_components: 2
+
+      - name: run application server
+        uses: appleboy/ssh-action@v1.1.0
+        with:
+          host: ${{ secrets.NCP_SERVER_HOST }}
+          username: ${{ secrets.NCP_SERVER_USERNAME }}
+          password: ${{ secrets.NCP_SERVER_PASSWORD }}
+          port: ${{ secrets.NCP_SERVER_PORT }}
+          script: |
+            docker-compose -f ${{ secrets.DOCKER_COMPOSE_YAML_PATH }} pull
+            docker-compose -f ${{ secrets.DOCKER_COMPOSE_YAML_PATH }} --env-file ${{ secrets.ENV_FILE_PATH }} up -d
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)